pyuvdata/uvdata/uvdata.py

```diff
@@ -2230,17 +2255,20 @@ def unphase_to_drift(self, phase_frame=None, use_ant_pos=False):
                     self.telescope_location_lat_lon_alt[0],
                     self.antenna_positions,
                 )
-
-                for bl_ind in inds:
-                    ant1_index = np.where(
-                        self.antenna_numbers == self.ant_1_array[bl_ind]
-                    )[0][0]
-                    ant2_index = np.where(
-                        self.antenna_numbers == self.ant_2_array[bl_ind]
-                    )[0][0]
-                    self.uvw_array[bl_ind, :] = (
-                        ant_uvw[ant2_index, :] - ant_uvw[ant1_index, :]
-                    )
+                # instead of looping through every ind, find the spot in antenna number
+                # array where ant_num <= ant1 < ant_number and similary for ant2
+                # for all baselines in inds
+                # then find the uvw coordinate for all at the same time
```

Comment:
This comment makes it sound like the antenna numbers need to be in order, but I don't think that's the case because you do an argsort below. Maybe just update the comment.

Reply:
I updated the comments, hopefully they are clearer now.